### ISSUE TITLE
add to <head>:  <link rel="canonical" href="lbry://[channel_ID]/[content_ID]">

### ISSUE THREAD

**Issue by ParaplegicRacehorse** [kodxana/madiator.com#5]
**Is your feature request related to a problem? Please describe.**
Is it a problem?  Hmm.... no.

**Describe the solution you'd like**
I want search-engine bots to know that this is not web-first content.

**Additional context**
Screenshot of Firefox browser developer tools output of a madiator .com content page.  Note there is not a rel="canonical" reference.  (Also note that the CSS declarations could maybe be cleaned up into a single call to a single CSS file?)

![2022-03-05_09-54=madiator-header](https://user-images.githubusercontent.com/7605896/156896584-fac04a12-250a-4bbb-8127-9aeb87e2c560.png)

**Comment by kodxana** [comment]
Umm do not get it.

**Comment by ParaplegicRacehorse** [comment]
Currently rel="canonical" tag points to web content (presumably, it changes based on instance config?).

LBRY content is not web content.  It is LBRY content.  It is not natively hyper-text transport protocol (http://) but is, rather, lbry (lbry://) protocol.

The canonical link type tells bots and browsers what is the primary-source of the content.  It is also pretty important for things like the AndroidOS "intents" system that allows some link types to be opened in their specialty apps rather than the browser.

**Comment by kodxana** [comment]
madiator.com is web based frontend and it's not using directly P2P protocol.

**Comment by ParaplegicRacehorse** [comment]
Right.  It is a web-based front-end, and not the authoritative content source.  Therefore, it should reference and point to the authoritative content source.
